Brain/brain_utils.h

```diff
@@ -66,6 +38,11 @@ String make_packet(struct sensorInfo sensor) {
     packet_content.remove(packet_content.length()-1); 
     int count = packet_content.length();
     char const *data = packet_content.c_str();
+    // c_str converts the String to it's underlying c-string type array and returns a
+    // const char* to avoid tampering of the data. The toCharArray makes a copy of the 
+    // string and you have to specify a length of the elements copied. Previously may not
+    // have been working due to length variations or possibly incorrect copying, not enough
+    // space to copy.
     uint16_t checksum = Fletcher16((uint8_t *) data, count);
     packet_content += "|";
```

Comment:
So, re c_str(), the issue we had was w/ `decode_received_packet`, not `make_packet`
If you take a look at the E1 branch, this func is very very similar, but the decode func is heavily modified. 
And we're decoding the checksum with c-str / toCharArray, so it's always constant size, so I'm not sure this is the issue.